docs/resources/logdna_view.md

```diff
@@ -25,12 +25,12 @@ provider "logdna" {
 }
 
 resource "logdna_view" "my_view" {
-  name     = "Terraform Alert Specific View Example"
+  name     = "Terraform View-Based Alert Example"
   query    = "my query"
   apps     = ["app1", "app2"]
   levels   = ["warn", "error"]
   hosts    = ["host1"]
-  category = ["Demo"]
+  categories = ["Demo1", "Demo2"]
```

Comment:
Lowercase on demo or upper-case on demo? Also what about apps, levels, hosts, query?

Reply:
I think this capitalization is good 👍